### PR TITLE
updates to readme, add default storagepool for jiva, updates jiva get response, updates jiva image to 0.6.0

### PR DESCRIPTION
**Pull request by AmitKumarDas**
This PR makes below changes:

- updates the README to make use of `openebs.github.io/charts` links
- updates jiva cas template to make use of jiva image 0.6.0
- adds a default storagepool that can be used to provision jiva volumes
- updates the get jiva volume's response with targetportal, iqn & replicas
- suffixes cstor volume target deployment with `-target`
- removes deprecated volumepolicies.yaml file

Signed-off-by: AmitKumarDas <amit.das@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by kmova** [comment]
@AmitKumarDas I didn't know we were linking to these files from our readme. We should change them to released versions.  
- https://openebs.github.io/charts/openebs-operator.yaml
- https://openebs.github.io/charts/openebs-storageclasses.yaml

Let us not mention about CAS templates till we release it. Also these steps will be simplified by the time we release 0.7.

File: README.md [hunk 1]
```diff
@@ -27,22 +27,24 @@ OpenEBS can scale to include an arbitrarily large number of containerized storag
  
 OpenEBS can be setup in few easy steps. You can get going on your choice of Kubernetes cluster by having open-iscsi installed on the Kubernetes nodes and running the openebs-operator using kubectl.
```
Comment (mahebbar) [inline README.md]: You can setup OpenEBS in a few easy steps.

File: README.md [hunk 2]
```diff
@@ -27,22 +27,24 @@ OpenEBS can scale to include an arbitrarily large number of containerized storag
...
 ```
+
 You could also follow our [QuickStart Guide](https://docs.openebs.io/docs/overview.html).
 
-OpenEBS can be deployed on any Kubernetes cluster - either in cloud, on-premise or developer laptop (minikube). Please follow our [OpenEBS Setup](https://docs.openebs.io/docs/overview.html) documentation. Also, we have a Vagrant environment available that includes a sample Kubernetes deployment and synthetic load that you can use to simulate the performance of OpenEBS. You may also find interesting the related project called Litmus (https://www.openebs.io/litmus) which helps with chaos engineering for stateful workloads on Kubernetes.  
+OpenEBS can be deployed on any Kubernetes cluster - either in cloud, on-premise or developer laptop (minikube). Please follow our [OpenEBS Setup](https://docs.openebs.io/docs/overview.html) documentation. Also, we have a Vagrant environment available that includes a sample Kubernetes deployment and synthetic load that you can use to simulate the performance of OpenEBS. You may also find interesting the related project called Litmus (https://www.openebs.io/litmus) which helps with chaos engineering for stateful workloads on Kubernetes.
```
Comment (mahebbar) [inline README.md]: You may also find interesting the related project called Litmus = You may also find the related project called Litmus interesting.

File: README.md [hunk 3]
```diff
@@ -27,22 +27,24 @@ OpenEBS can scale to include an arbitrarily large number of containerized storag
...
 
 ## Status
-We are approaching beta stage with active development underway. See our [Project Tracker](https://github.com/openebs/openebs/wiki/Project-Tracker) for more details.  Please note that as of June 2018 we have cStor which is about to be released for additional funcitons and somewhat improved performance; please contact us on Slack if you are interested in this engine.  
+We are approaching beta stage with active development underway. See our [Project Tracker](https://github.com/openebs/openebs/wiki/Project-Tracker) for more details.  Please note that as of June 2018 we have cStor which is about to be released for additional funcitons and somewhat improved performance; please contact us on Slack if you are interested in this engine.
```
Comment (mahebbar) [inline README.md]: funcitons = functions
and somewhat = with improved...

**Pull request merged**